lib/libzfsacl/zfsacltests/test_nfsv4acl.py

```diff
@@ -0,0 +1,1531 @@
+import unittest
+import os
+import pwd
+import shutil
+import libzfsacl
+import sys
+from subprocess import run, PIPE
+
+
+def run_as_user(cmd, user):
+    if shutil.which(cmd.split()[0]) is not None:
+        cmd = shutil.which(cmd.split()[0]) + " " + " ".join(cmd.split()[1:])
+    command = ["/usr/bin/su", "-", user, "-c", cmd]
+    proc = run(command, stdout=PIPE, stderr=PIPE,
+               universal_newlines=True, timeout=30)
+    if proc.returncode != 0:
+        return {"result": False, "output": proc.stdout,
+                "error": proc.stderr, "returncode": proc.returncode}
+    else:
+        return {"result": True, "output": proc.stdout,
+                "error": proc.stderr, "returncode": proc.returncode}
+
+
+class TestNFSAcl(unittest.TestCase):
+
+    ZFS_ACL_STAFF_GROUP = "zfsgrp"
+    ZFS_ACL_STAFF1 = "staff1"
+    ZFS_ACL_STAFF2 = "staff2"
+    ZFS_ACL_STAFF1_UID = 0
+    ZFS_ACL_STAFF2_UID = 0
+    MOUNTPT = "/var/tmp/testdir"
+    TESTPOOL = "testpool"
+    TESTFS = "testfs"
+    TDIR = '/var/tmp/testdir/test'
```

Comment:
We are hardcoding these, if it's easily doable, can we utilize these values from ZTS?

Reply:
I tried doing that previously, but unfortunately did not find a way to do that.